db/database_test.go

```diff
@@ -674,28 +674,28 @@ func TestAllDocsOnly(t *testing.T) {
 	assert.Equal(t, 100, len(changes))
 
 	for i, change := range changes {
-		seq := i + 1
+		seq := i
 		if i >= 23 {
 			seq++
 		}
-		assert.Equal(t, SequenceID{Seq: uint64(seq)}, change.Seq)
+		assert.Equal(t, i != 99, fmt.Sprintf("alldoc-%02d", seq) == change.ID)
```

Comment:
The way this section is written (both in the original test and now) is hard to follow.  I think it would be better written as:
```suggestion
                 if i == len(changes) - 1 {
                    // The last entry in the changes response should be the deleted document
                    assert.True(t, change.Deleted)
                    assert.Equal(t, "alldoc-23", change.ID)
                    assert.Equal(t, channels.SetOf(t, "all"), change.Removed)
                 } else {
                    // Verify correct ordering for all other documents
                    assert.Equal(t, fmt.Sprintf("alldoc-%02d", docIndex), change.ID)
                 }
```

Reply:
Suggested change has been committed to latest revision.